tools/run_tests/run_interop_tests.py

```diff
@@ -799,14 +807,19 @@ def cloud_to_cloud_jobspec(language,
                            server_host,
                            server_port,
                            docker_image=None,
-                           insecure=False,
+                           transport_security='tls',
                            manual_cmd_log=None):
     """Creates jobspec for cloud-to-cloud interop test"""
     interop_only_options = [
         '--server_host_override=foo.test.google.fr',
-        '--use_tls=%s' % ('false' if insecure else 'true'),
         '--use_test_ca=true',
     ]
+    if transport_security == 'tls':
+        interop_only_options += ['--use_tls=true']
+    elif transport_security == 'alts':
+        interop_only_options += ['--use_tls=false', '--use_alts=true']
+    elif transport_security == 'none':
+        interop_only_options += ['--use_tls=false']
```

Comment:
robustness: if none of the options is matched, this should throw. Silently ignoring the problem is dangerous.